base/monit.supervisord.conf

```diff
@@ -0,0 +1,5 @@
+[program:monit]
```

Comment:
Does monit actually run correctly under Docker?  Last time I checked, it didn't (I think because there was no init process, and so nothing to catch zombie processes, but it's been a while so this could be wrong).  I don't think changing this is something we should undertake lightly - let's discuss.